build.sh

```diff
@@ -184,6 +186,10 @@ while [[ $# -gt 0 ]]; do
         ICU_PATH="-DNO_ICU_PATH_GIVEN_SH=1"
         ;;
 
+    --no-jit)
+        NO_JIT="-DNO_JIT=1"
```

Comment:
nit:`_SH` suffix is useful for remembering the source of the variable though. The rest of the definitions from build.sh use the same suffix.